src/ResponseCache.php

```diff
@@ -84,13 +85,19 @@ public function getCachedResponseFor(Request $request, array $tags = []): Respon
         return $this->taggedCache($tags)->get($this->hasher->getHashFor($request));
     }
 
-    public function clear(array $tags = []): void
+    public function clear(array $tags = []): bool
     {
         event(new ClearingResponseCache());
 
-        $this->taggedCache($tags)->clear();
+        $result = $this->taggedCache($tags)->clear();
 
-        event(new ClearedResponseCache());
+        if ($result === true) {
```

Comment:
Avoid using `else` by refactoring to a ternary to determine the event classname.

Reply:
done by https://github.com/spatie/laravel-responsecache/pull/495/commits/f63a605018593ad5d42bac407c31d0887aa1e8bf